src/main/java/com/controller/QuoteController.java

```diff
@@ -128,4 +134,28 @@ public ResponseEntity dislikeQuote(@PathVariable int id) {
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
+    @GetMapping(path = "/elk")
+    public ResponseEntity generateLog() {
+        String response = "Generating log " + new Date();
+        LOG.info(response);
+        LOG.debug(response);
+        LOG.error(response);
+        return new ResponseEntity<>(response, HttpStatus.OK);
+    }
+
+
+    @GetMapping(path = "/exception")
+    public ResponseEntity error() {
+        String errorResponse = "";
+        try {
+            throw new Exception("Test exception");
```

Comment:
if this is a test, move it to the tests

Reply:
Integration tests will be implemented at a later date.